hcx-apis/src/main/java/org/swasth/hcx/controllers/v1/ParticipantController.java

```diff
@@ -113,4 +111,12 @@ private ParticipantResponse errorResponse(ErrorCodes code, String message, Throw
         return resp;
     }
 
+    private HttpEntity<ParticipantResponse> validateParticipant (Map<String, Object> body) {
+        List<String> notAllowedUrls = env.getProperty(HCX_NOT_ALLOWED_URLS, List.class, new ArrayList<String>());
```

Comment:
Remove return null and add method return type to void

Reply:
while using new ResponseEntity<> inside method unable to change method name void.

Reply:
Ok